.github/workflows/release.sh

```diff
@@ -220,13 +223,22 @@ cd_kerl_dir
 kerl_build_install "${global_OTP_VSN}"
 echo "::endgroup::"
 
-kerl_test() {
+erl_test() {
     ./bin/erl -s crypto -s init stop
     ./bin/erl_call
+    local crypto
+    crypto=$(find . -name "crypto.so")
```

Comment:
FWIW,  a simpler method here might be `dyld_info "${crypto}" | grep -q openssl`, if the match, the exit status will be zero, otherwise 1.

Reply:
It's a minor detail, I guess (?) or is this what prevented the approval?

Reply:
It is minor, it was merely a suggestion, but not a blocker.